webpack.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
     new webpack.DefinePlugin({
       'process.env.NODE_ENV': JSON.stringify('staging')
     }),
-    new DashboardPlugin({ port: 3001 })
+    new DashboardPlugin({ port: 3001 }) // Change this here and in the package.json start script if needed
```

Comment:
This LGTM, but just a semantic question: why do you think "if needed" is necessary in this comment? Seems superfluous to me, given that the docs say that it needs to be added in both places.